gene2phenotype_project/gene2phenotype_app/models.py

```diff
@@ -481,6 +483,7 @@ class Panel(models.Model):
     name = models.CharField(max_length=100, unique=True, null=False)
     description = models.CharField(max_length=255, null=True)
     is_visible = models.SmallIntegerField(null=False)
+    stats = models.JSONField(null=True, default=None)
```

Comment:
The code to calculate the stats has been improved in [this PR](https://github.com/EBI-G2P/gene2phenotype_api/pull/16) which means we don't need to save the stats in the db.